src/localize/languages/pl.json

```
@@ -15,6 +15,8 @@
       "close_cover": "zamknij",
       "open_cover": "otwórz",
       "set_cover_position": "ustaw pozycję[ na {position}]"

```

The JSON is invalid.
```suggestion
      "set_cover_position": "ustaw pozycję[ na {position}]",
```